platforms/BUILD

```diff
@@ -14,31 +17,40 @@ platform(
     ],
     exec_properties = {
         "OSFamily": "Linux",
-        "container-image": "docker://gcr.io/flame-public/rbe-ubuntu20-04:latest",
         "dockerNetwork": "off",
     },
 )
 
 platform(
-    name = "macos_x86_64",
-    constraint_values = [
-        "@platforms//cpu:x86_64",
-        "@platforms//os:macos",
-    ],
+    name = "linux_x86_64",
+    constraint_values = ["@platforms//cpu:x86_64"],
+    exec_properties = {"container-image": "docker://" + UBUNTU20_04_IMAGE},
+    parents = [":linux"],
+)
+
+platform(
+    name = "macos",
+    constraint_values = ["@platforms//os:macos"],
     exec_properties = {
         "Arch": "amd64",
         "OSFamily": "darwin",
     },
 )
 
+platform(
+    name = "macos_x86_64",
+    constraint_values = ["@platforms//cpu:x86_64"],
+    parents = [":macos"],
+)
+
 platform(
     name = "macos_arm64",
-    constraint_values = [
-        "@platforms//cpu:arm64",
-        "@platforms//os:macos",
-    ],
-    exec_properties = {
-        "Arch": "arm64",
-        "OSFamily": "darwin",
-    },
+    constraint_values = ["@platforms//cpu:arm64"],
+    parents = [":macos"],
+)
+
+platform(
+    name = "local_config_platform",
```

Comment:
Nit: don’t need to address this now but we should discuss when/how this could be used later for a potential better name for this. It’s fairly similar to `@local_config_platform//:host` with very subtle differences.

Reply:
Sure, I named it that because it uses the `@local_config_platform` "package" to fill in its values, but happy to rename it when we upstream this stuff to buildbuddy-toolchain.